core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties

```diff
@@ -0,0 +1,25 @@
+pleaseRekeyAsap=\
+  Из-за <a href="https://www.jenkins.io/redirect/rekey" rel="noopener noreferrer" target="_blank">уязвимости</a>, \
+  обнаруженной ранее, необходимо \
+  сменить ключ шифрования, используемый для защиты секретов в файлах конфигурации на диске. \
+  Этот процесс просканирует большую часть директории <code>$JENKINS_HOME</code> (<code>{0}</code>), \
+  найдёт шифрованные данные и сменит ключ. Это займёт некоторое время. \
+  Больше о результатах разных способов это сделать (или этого не делать) \
```

Comment:
`См. этот документ для получения дополнительной информации о последствиях этой операции (или какими могут быть последствия бездействия).`

Fine with tweaking few words here and there if you wish. The message should stay though.

Reply:
Looking at the link and how far in the past it has happened I don't thinking anyone will ever see this translation.

Reply:
this is what I mean by:
> Some messages are potentially outdated (Java 11, security issues) but even so they could then be a base for a similar change message in the future.
if there is ever another problem like that that'd require a similar operation this could be used (or if someone hasn't used this Jenkins for a very long time 😆 )